calendar-bundle/src/Resources/contao/classes/Events.php

```diff
@@ -579,6 +579,35 @@ protected function getDatesFromFormat(Date $objDate, $strFormat)
 				return array(0, time(), $GLOBALS['TL_LANG']['MSC']['cal_empty']);
 		}
 	}
+
+	public static function getBasicSchemaOrgData(CalendarEventsModel $objEvent): array
```

Comment:
Why did you add this method but not an equivalent `News::getBasicSchemaOrgData()` method? In the `ModuleNews` class, this logic only exists in the anonymous function:

https://github.com/contao/contao/blob/829cbf481d20962388a5b0fce446c64c2a948e21/news-bundle/src/Resources/contao/modules/ModuleNews.php#L231-L260

Reply:
Because unfortunately the ModuleCalendarReader does not share any logic. In the news bundle we had a central `parseArticle()` which was reused in the reader as well. Here we don't have that, hence the new helper method.

Reply:
Would you be OK with me renaming the method to `getSchemaOrgDataFromEvent()` and adding an equivalent method `getSchemaOrgDataFromNews()` to the `News` class? The helper methods might be useful in other cases, too.

Reply:
NVM, I will create a separate PR hereto.

Reply:
Yeah, I think for the sake of consistency that would make sense 👍 But can also be a PR after this one, sure :)

Reply:
See #3161